src/components/draft-js-editor/__tests__/DraftJSEditor.test.js

```diff
@@ -23,22 +23,13 @@ describe('components/draft-js-editor/DraftJSEditor', () => {
     describe('render()', () => {
         test('should correctly render the component', () => {
             const wrapper = shallow(<DraftJSEditor {...requiredProps} />);
-            const label = wrapper.find('Label');
             const tooltip = wrapper.find('Tooltip');
             expect(wrapper.hasClass('draft-js-editor')).toBe(true);
-
-            expect(label.length).toBe(1);
-            expect(label.prop('hideLabel')).toBeFalsy();
             expect(tooltip.length).toBe(1);
             expect(tooltip.children().is('div')).toBe(true);
             expect(wrapper.find(Editor).length).toBe(1);
         });
 
-        test('should hide label when specified', () => {
-            const wrapper = shallow(<DraftJSEditor {...requiredProps} hideLabel />);
-            expect(wrapper.find('Label').prop('hideLabel')).toBe(true);
```

Comment:
We should change this test to look for the `accessibility-hidden` class on the label span

Reply:
👍